runtime/src/lib.rs

```diff
@@ -260,6 +261,67 @@ impl pallet_evm::Config for Runtime {
 	type ChainId = EthereumChainId;
 }
 
```

Comment:
Can @JoshOrndorff or @4meta5 look at that configuration for now. Just to double check there is no obvious misconfiguration.

Reply:
Looks like a good place to start testing to me.